### DISCUSSION TITLE
Add dedicated `bool` column to identify whether a dashboard is an Example.

### DISCUSSION THREAD

**Discussion by EBoisseauSierra** [apache/superset#18528] [Ideas]
**Problem**

I would like to be able to customize which dashboards appear in the “Example” tab of my Superset instance.

**Describe the solution you'd like**

Currently, Example dashboards are those which `created_by_fk` value is `NULL` (cf. [Slack discussion](https://apache-superset.slack.com/archives/C0170U650CQ/p1636630702324500)).

A first step towards exposing the “Example” status of a dashboard (to the CLI or GUI) would be to create a dedicated additional `bool` column to the `dashboards` table — say `is_example`.

**Describe alternatives you've considered**

* Create a new *table* to track the ID of the examples dashboards… but doesn't quite sound like a good idea. 
* Use the dashboard's JSON metadata… but we might not want to allow every users having this permission to brand their dashboard as “Example”.

**Comment by nytai** [comment]
Adding some thoughts there:

Once the bool values is added to the dashboard table a couple updates will be required to support the current behavior.
- The load-examples script will have to add the is_examples attr to the generated dashboard
- the home screen api request for the examples call will have to be updated to filter on the is_example attr

Some additional thoughts for this feature:
- I would be great to separate out these dashboards into another section in the dashboards list and/or
- have a filter for this attribute

**Comment by EBoisseauSierra** [comment]
As @nytai also mentioned on Slack, we might want to go one step further and generalize beyond the Examples dashboards only, by considering sets of curated dashboards (that Examples would only be an instance thereof).
